dbinit/init.sql

```diff
@@ -51,7 +51,8 @@ create table users (
     -- email is coming from openid registration.
     email varchar(256) unique not null,
     last_login date,
-    display_name varchar(50)
+    display_name varchar(50),
+    token varchar(60) unique
```

Comment:
unique not null
So we don't insert a user by accident without a token?

Reply:
So I thought we'd want a user to only generate a token if they planned to use it (rather than create one when we create the user).

I believe that in Postgres NULL is treated as distinct. So this just means that we can't have duplicate tokens once generated.

Reply:
True. Okay.